core-web/package.json

```diff
@@ -238,6 +238,7 @@
         "jest": "29.7.0",
         "jest-cli": "29.7.0",
         "jest-environment-jsdom": "29.7.0",
+        "jest-environment-node": "^29.7.0",
```

Comment:
Why?

Reply:
Not sure, this happened when I created the library

Reply:
It was a configuration error. Im deleting this line